docs/reference/platforms/unix-prerequisites.rst

```diff
@@ -2,34 +2,46 @@ These instructions are different on almost every version of Linux and Unix; here
 some of the common alternatives:
 
 ..
-    The package list should be the same as in ci.yml, and the BeeWare tutorial.
+    The package list should be the same as in ci.yml, and the BeeWare tutorial
+    (CI will also have WebView requirements)
 
-**Ubuntu 18.04+ / Debian 10+**
+**Ubuntu 18.04, 20.04 / Debian 10**
 
 .. code-block:: console
 
     (venv) $ sudo apt update
-    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev gir1.2-webkit2-4.0 libcanberra-gtk3-module
+    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
+
+**Ubuntu 22.04+ / Debian 11+**
+
+.. code-block:: console
+
+    (venv) $ sudo apt update
+    (venv) $ sudo apt install pkg-config python3-dev libgirepository1.0-dev libcairo2-dev libcanberra-gtk3-module
```

Comment:
Do we need separate sections? They're the same install instructions regardless of version.

Reply:
That's true - my first pass still had the WebKit2 install details, but we can simplify now they've been offloaded to the widget.